instrumented/Components/EmailNotificationForm/states.js

```diff
@@ -0,0 +1,204 @@
ICB7IHZhbHVlOiBcImF6XCIsIGxhYmVsOiBcIkFyaXpvbmFcIiB9LFxuICAgIHsgdmFsdWU6IFwiYXJcIiwgbGFiZWw6IFwiQXJrYW5zYXNcIiB9LFxuICAgIHsgdmFsdWU6IFwiY2FcIiwgbGFiZWw6IFwiQ2FsaWZvcm5pYVwiIH0sXG4gICAgeyB2YWx1ZTogXCJjb1wiLCBsYWJlbDogXCJDb2xvcmFkb1wiIH0sXG4gICAgeyB2YWx1ZTogXCJjdFwiLCBsYWJlbDogXCJDb25uZWN0aWN1dFwiIH0sXG4gICAgeyB2YWx1ZTogXCJkZVwiLCBsYWJlbDogXCJEZWxhd2FyZVwiIH0sXG4gICAgeyB2YWx1ZTogXCJkY1wiLCBsYWJlbDogXCJEaXN0cmljdCBPZiBDb2x1bWJpYVwiIH0sXG4gICAgeyB2YWx1ZTogXCJmbFwiLCBsYWJlbDogXCJGbG9yaWRhXCIgfSxcbiAgICB7IHZhbHVlOiBcImdhXCIsIGxhYmVsOiBcIkdlb3JnaWFcIiB9LFxuICAgIHsgdmFsdWU6IFwiaGlcIiwgbGFiZWw6IFwiSGF3YWlpXCIgfSxcbiAgICB7IHZhbHVlOiBcImlkXCIsIGxhYmVsOiBcIklkYWhvXCIgfSxcbiAgICB7IHZhbHVlOiBcImlsXCIsIGxhYmVsOiBcIklsbGlub2lzXCIgfSxcbiAgICB7IHZhbHVlOiBcImluXCIsIGxhYmVsOiBcIkluZGlhbmFcIiB9LFxuICAgIHsgdmFsdWU6IFwiaWFcIiwgbGFiZWw6IFwiSW93YVwiIH0sXG4gICAgeyB2YWx1ZTogXCJrc1wiLCBsYWJlbDogXCJLYW5zYXNcIiB9LFxuICAgIHsgdmFsdWU6IFwia3lcIiwgbGFiZWw6IFwiS2VudHVja3lcIiB9LFxuICAgIHsgdmFsdWU6IFwibGFcIiwgbGFiZWw6IFwiTG91aXNpYW5hXCIgfSxcbiAgICB7IHZhbHVlOiBcIm1lXCIsIGxhYmVsOiBcIk1haW5lXCIgfSxcbiAgICB7IHZhbHVlOiBcIm1kXCIsIGxhYmVsOiBcIk1hcnlsYW5kXCIgfSxcbiAgICB7IHZhbHVlOiBcIm1hXCIsIGxhYmVsOiBcIk1hc3NhY2h1c2V0dHNcIiB9LFxuICAgIHsgdmFsdWU6IFwibWlcIiwgbGFiZWw6IFwiTWljaGlnYW5cIiB9LFxuICAgIHsgdmFsdWU6IFwibW5cIiwgbGFiZWw6IFwiTWlubmVzb3RhXCIgfSxcbiAgICB7IHZhbHVlOiBcIm1zXCIsIGxhYmVsOiBcIk1pc3Npc3NpcHBpXCIgfSxcbiAgICB7IHZhbHVlOiBcIm1vXCIsIGxhYmVsOiBcIk1pc3NvdXJpXCIgfSxcbiAgICB7IHZhbHVlOiBcIm10XCIsIGxhYmVsOiBcIk1vbnRhbmFcIiB9LFxuICAgIHsgdmFsdWU6IFwibmVcIiwgbGFiZWw6IFwiTmVicmFza2FcIiB9LFxuICAgIHsgdmFsdWU6IFwibnZcIiwgbGFiZWw6IFwiTmV2YWRhXCIgfSxcbiAgICB7IHZhbHVlOiBcIm5oXCIsIGxhYmVsOiBcIk5ldyBIYW1wc2hpcmVcIiB9LFxuICAgIHsgdmFsdWU6IFwibmpcIiwgbGFiZWw6IFwiTmV3IEplcnNleVwiIH0sXG4gICAgeyB2YWx1ZTogXCJubVwiLCBsYWJlbDogXCJOZXcgTWV4aWNvXCIgfSxcbiAgICB7IHZhbHVlOiBcIm55XCIsIGxhYmVsOiBcIk5ldyBZb3JrXCIgfSxcbiAgICB7IHZhbHVlOiBcIm5jXCIsIGxhYmVsOiBcIk5vcnRoIENhcm9saW5hXCIgfSxcbiAgICB7IHZhbHVlOiBcIm5kXCIsIGxhYmVsOiBcIk5vcnRoIERha290YVwiIH0sXG4gICAgeyB2YWx1ZTogXCJvaFwiLCBsYWJlbDogXCJPaGlvXCIgfSxcbiAgICB7IHZhbHVlOiBcIm9rXCIsIGxhYmVsOiBcIk9rbGFob21hXCIgfSxcbiAgICB7IHZhbHVlOiBcIm9yXCIsIGxhYmVsOiBcIk9yZWdvblwiIH0sXG4gICAgeyB2YWx1ZTogXCJwYVwiLCBsYWJlbDogXCJQZW5uc3lsdmFuaWFcIiB9LFxuICAgIHsgdmFsdWU6IFwicmlcIiwgbGFiZWw6IFwiUmhvZGUgSXNsYW5kXCIgfSxcbiAgICB7IHZhbHVlOiBcInNjXCIsIGxhYmVsOiBcIlNvdXRoIENhcm9saW5hXCIgfSxcbiAgICB7IHZhbHVlOiBcInNkXCIsIGxhYmVsOiBcIlNvdXRoIERha290YVwiIH0sXG4gICAgeyB2YWx1ZTogXCJ0blwiLCBsYWJlbDogXCJUZW5uZXNzZWVcIiB9LFxuICAgIHsgdmFsdWU6IFwidHhcIiwgbGFiZWw6IFwiVGV4YXNcIiB9LFxuICAgIHsgdmFsdWU6IFwidXRcIiwgbGFiZWw6IFwiVXRhaFwiIH0sXG4gICAgeyB2YWx1ZTogXCJ2dFwiLCBsYWJlbDogXCJWZXJtb250XCIgfSxcbiAgICB7IHZhbHVlOiBcInZhXCIsIGxhYmVsOiBcIlZpcmdpbmlhXCIgfSxcbiAgICB7IHZhbHVlOiBcIndhXCIsIGxhYmVsOiBcIldhc2hpbmd0b25cIiB9LFxuICAgIHsgdmFsdWU6IFwid3ZcIiwgbGFiZWw6IFwiV2VzdCBWaXJnaW5pYVwiIH0sXG4gICAgeyB2YWx1ZTogXCJ3aVwiLCBsYWJlbDogXCJXaXNjb25zaW5cIiB9LFxuICAgIHsgdmFsdWU6IFwid3lcIiwgbGFiZWw6IFwiV3lvbWluZ1wiIH1cbiAgXVxufSJdfQ==
```

Comment:
I didn't see this file being used anywhere in the tests. Can you please explain what this file is for and if it's needed?

Reply:
'Explained' [here](https://github.com/mi-voz-mi-voto/mi-voz-mi-voto/pull/30#discussion_r804866273)